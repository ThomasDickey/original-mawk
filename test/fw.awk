#
# basic tests of FIELDWIDTHS setting of fixed width fields
#
BEGIN {
  FIELDWIDTHS="2 14 10 3"
  OFS="~~"
}
{ 
  print $2,$1,$4,$3; 
  FS=FS
}
