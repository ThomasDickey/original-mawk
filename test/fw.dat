The quick brown fox jumped over the lazy dog 0123456789
The quick brown fox jumped over the lazy dog 0123456789
