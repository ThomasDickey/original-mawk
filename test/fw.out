e quick brown ~~Th~~ ov~~fox jumped
quick~~The~~fox~~brown
